src/main/kotlin/com/openlattice/authorization/processors/AuthorizationEntryProcessor.kt

```diff
@@ -0,0 +1,15 @@
+package com.openlattice.authorization.processors
+
+import com.openlattice.authorization.AceKey
+import com.openlattice.authorization.AceValue
+import com.openlattice.authorization.DelegatedPermissionEnumSet
+import com.openlattice.authorization.Permission
+import com.openlattice.rhizome.hazelcast.entryprocessors.AbstractReadOnlyRhizomeEntryProcessor
+import java.util.*
+
+class AuthorizationEntryProcessor : AbstractReadOnlyRhizomeEntryProcessor<AceKey, AceValue, DelegatedPermissionEnumSet>() {
```

Comment:
I am unsure but this may also be `Offloadable`

Reply:
I thought it inherits that from `AbstractReadOnlyRhizomeEntryProcessor`? But I guess it only inherits `ReadOnly` now that I look at it